docs/source/configuration.rst

```diff
@@ -198,6 +198,14 @@ feature and you don't mind that your room bookmarks are visible to all
 contacts, then you can set this setting to ``true``. Otherwise you won't be
 able to have any room bookmarks at all for an account on that XMPP server.
 
+allow_occupants_view_resizing
+----------------------
+
+* Default: ``true``
+
+Allows users to resize the occupants-list inside multi-user-chats. Setting this to ``false`` 
+deactivates the resizing-capability.
+
```

Comment:
I'm not convinced that this config setting is necessary. We already have the `dragresize` config seting (which I also wonder whether it's really necessary), so perhaps we can just let that setting determine drag-resizing for the boxes and the sidebar.

Continuously adding new config options makes the code more complex and creates more places for bugs to hide, so I actually want to try and reduce the amount of config options, or at least keep them to a minimum.